tests/validator/input/api/error_definitionbody.yaml

```diff
@@ -0,0 +1,15 @@
+Transform: AWS::Serverless-2016-10-31
```

Comment:
Sorry if I asked this before and just don't remember. Is there a reason add all new `.yaml` files instead of bootstrapping off the list we already have? 

Just thinking about maintenance going forward. It's already a pain to add all the (what I call) functional tests that run the translator locally. My concern here is that we have to add a whole new set of files on top of the other ones we have. At first it feels like duplicated effort but could be a very good reason to do it this way over what I expected.

Reply:
This would have required to update all the output files for the affected translator files and would have made this PR even larger.

Some of the errors caught by the schema validation are also not caught by the translator and can lead to exceptions/crashes.

What we could consider however is to use the validator tests for the translator in a future PR.

Reply:
Maybe separate PRs? I am generally concerned that we duplicated tons of effort and makes it hard (that it is) to do these end to end style testing. 

Not blocking here on this, but I feel pretty strong and finding a way out of this in the future.